### PR TITLE
adding maven pom file

### PR DESCRIPTION
**Pull request by luxu1-ms**
Adding pom.xml to build maven integration for spark sql connector.

### CODE REVIEW

**Comment by shivsood** [comment]
Thanks @luxu1-ms. Can we also add the git actions integrations to push to maven as part of this PR. 
@rajmera3 can u help define release plan versioning and as for we do a manual monthly release push to maven. Thoughts?

**Comment by rajmera3** [comment]
@luxu1-ms 
https://docs.github.com/en/actions/language-and-framework-guides/publishing-java-packages-with-maven
https://github.com/actions/starter-workflows/blob/e11672e5a9026a98ce6235af089d3112ec93dbd5/ci/maven-publish.yml 

These links should help set up the Maven publish. Haven't tried it myself but seemed like thats what people recommend

**Comment by sandeep540** [comment]
Any ETA for this? We want to Use it

**Comment by rajmera3** [comment]
@luxu1-ms @shivsood If the pom file is correct, please go ahead and merge and publish to maven manually. We will automate this later.

**Comment by shivsood** [comment]
Tested and validated new jar on BDC

**Comment by shivsood** [comment]
@rajmera3 can u validate on DBR.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.sqlserver.jdbc.spark</groupId>
+    <artifactId>spark-mssql-connector_2.11</artifactId>
+    <packaging>jar</packaging>
+    <description>spark-mssql-connector</description>
+    <version>1.0.0</version>
+    <name>spark-mssql-connector</name>
```
Comment (rajmera3) [inline pom.xml:9]: @luxu1-ms If this is the name that the project will be listed by, please use this official name: Apache Spark Connector for SQL Server and Azure SQL

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.sqlserver.jdbc.spark</groupId>
+    <artifactId>spark-mssql-connector_2.11</artifactId>
+    <packaging>jar</packaging>
+    <description>spark-mssql-connector</description>
```
Comment (rajmera3) [inline pom.xml:7]: Please update the description: The Apache Spark Connector for SQL Server and Azure SQL is a high-performance connector that enables you to use transactional data in big data analytics and persists results for ad-hoc queries or reporting.

**Comment by luxu1-ms** [comment]
I wonder if the groupId is com.microsoft.azure?

**Comment by rajmera3** [comment]
> I wonder if the groupId is com.microsoft.azure?

I think thats fine. We can leave it there

**Comment by mattnischan** [comment]
@rajmera3 @luxu1-ms Did this get published manually as described? I'm unable to locate the package on Maven at the moment.

**Pull request merged**